iidm/iidm-converter-api/src/main/java/com/powsybl/iidm/IidmImportExportMode.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) 2019, All partners of the iTesla project (http://www.itesla-project.eu/consortium)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm;
+
+/***
+ * @author Chamseddine BENHAMED <chamseddine.benhamed at rte-france.com>
+ */
+
+public enum IidmImportExportMode {
```

Comment:
Rename to  IidmExportMode if mode move to ExportOptions?

Reply:
see first remark